build/study.js

```diff
@@ -553,7 +553,10 @@ var Study = function () {
   }, {
     key: 'assign',
     value: function assign(testName, bucketName) {
-      if (!testName) return this.assignAll();
+      if (!testName) {
```

Comment:
don't update the build file directly. we also want to check that the test name is a valid test, not just that it exists